code/game/machinery/OpTable.dm

```diff
@@ -43,60 +40,58 @@
 		return TRUE
 
 /obj/machinery/optable/CanPass(atom/movable/mover, turf/target, height=0)
-	if(height==0) return 1
-
+	if(height==0)
+		return TRUE
 	if(istype(mover) && mover.checkpass(PASSTABLE))
-		return 1
+		return TRUE
 	else
-		return 0
+		return FALSE
 
 
-/obj/machinery/optable/MouseDrop_T(atom/movable/O as mob|obj, mob/user as mob)
-	if(usr.stat || (!ishuman(user) && !isrobot(user)) || user.restrained() || !check_table(user) || user.IsWeakened() || user.stunned)
+/obj/machinery/optable/MouseDrop_T(atom/movable/O, mob/user)
+	if(!ishuman(user) && !isrobot(user)) //Only Humanoids and Cyborgs can put things on this table
 		return
-
-	if(!ismob(O)) //humans only
+	if(!check_table()) //If the Operating Table is occupied, you cannot put someone else on it
 		return
-
-	if(istype(O, /mob/living/simple_animal) || istype(O, /mob/living/silicon)) //animals and robots dont fit
+	if(user.restrained() || user.buckled || user.IsWeakened() || user.stunned || user.incapacitated()) //Is the person trying to use the table incapacitated or restrained?
```

Comment:
`user.incapacitated()` will already check most of these. Only the buckled check is not done there.

Reply:
There was confusion between a few of us on this. Thanks for clearing this up!